src/promptflow/promptflow/_sdk/operations/_run_operations.py

```diff
@@ -138,13 +140,20 @@ def stream(self, name: Union[str, Run]) -> Run:
             available_logs = local_storage.logger.get_logs()
             incremental_print(available_logs, printed, file_handler)
             self._print_run_summary(run)
-            # print error message when run is failed
-            if run.status == RunStatus.FAILED:
-                error_message = local_storage.load_exception()["message"]
-                print_red_error(error_message)
         except KeyboardInterrupt:
             error_message = "The output streaming for the run was interrupted, but the run is still executing."
             print(error_message)
+
+        if run.status == RunStatus.FAILED or run.status == RunStatus.CANCELED:
```

Comment:
Nit: can we share the logic between pfazure & pf?

Reply:
there are different ways to get error message for pf and pfazure, so I prefer not share them for now.